tracer/src/Datadog.Trace/Iast/IastModule.cs

```diff
@@ -390,6 +389,20 @@ public static IastModuleResponse OnInsecureAuthProtocol(string authHeader, Integ
         return AddWebVulnerability(authHeader, integrationId, VulnerabilityTypeName.InsecureAuthProtocol, (VulnerabilityTypeName.InsecureAuthProtocol + ':' + authHeader).GetStaticHashCode());
     }
 
+    public static void OnDirectoryListingLeak()
+    {
+        if (!Iast.Instance.Settings.Enabled) { return; }
+
+        var vulnerability = new Vulnerability(
+            VulnerabilityTypeName.DirectoryListingLeak,
+            VulnerabilityTypeName.DirectoryListingLeak.GetStaticHashCode(),
+            GetLocation(),
+            new Evidence("Directory listing is configured"),
```

Comment:
Is there a way of knowing why is it enabled or where is it configured, in order to give more insights to the user about how to fix it?

Reply:
Isn't this information provided inside the UI with the remediation text?
The precise location is still given if the app have symbols.